tests/integration_tests/test_experiment.py

```diff
@@ -477,6 +478,87 @@ def test_visual_question_answering(csv_filename):
     os.rmdir(image_dest_folder)
 
 
+def test_image_resizing_num_channel_handling(csv_filename):
```

Comment:
instead of adding a whole new integration test (which are pretty slow) can we add a couple new features to one of the existing integration tests involving images?

Reply:
Yes the tests are slow. But the tricky part here is that I have to generate two datasets with different number of channels. We can eliminate some of the image related tests because of this. Maybe I can do that as part of another task?

Reply:
sg